adapters/repos/db/vector/dynamic/index.go

```diff
@@ -49,9 +49,13 @@ var dynamicBucket = []byte("dynamic")
 type VectorIndex interface {
 	Dump(labels ...string)
 	Add(ctx context.Context, id uint64, vector []float32) error
+	AddMulti(ctx context.Context, docId uint64, vector [][]float32) error
```

Comment:
suggestion: Maybe here we could also introduce new `type VectorMultiIndex interface` and embed it in `VectorIndex`